src/beautifiers/brittany.coffee

```diff
@@ -0,0 +1,24 @@
+###
+Requires https://github.com/lspitzner/brittany
+###
+
+"use strict"
+Beautifier = require('./beautifier')
+
+module.exports = class Brittany extends Beautifier
+  name: "brittany"
+  link: "https://github.com/lspitzner/brittany"
+  isPreInstalled: false
+
+  options: {
+    Haskell: true
```

Comment:
`true` indicates all options are supported. Looking at code below, this should be `false` indicating language is supported with no supported options.